src/cocotb_tools/runner.py

```diff
@@ -488,13 +507,13 @@ def get_results(results_xml_file: Path) -> Tuple[int, int]:
         Tuple of number of tests and number of fails.
 
     Raises:
```

Comment:
Can this also raise on parsing the file? Is that a problem?

Reply:
It can, but it isn't an expected failure path, so we should just let it rise.